pro_tes/api/openapi.yaml

```diff
@@ -0,0 +1,611 @@
+openapi: 3.0.0
```

Comment:
Please indicate the exact commit hash (short form of 7 letters) in the file name, exactly like in proWES. And just to be clear: please do never edit this file manually. It should be _exactly_ identical to the file on the TES repo, for the indicated commit.